GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1665,7 +1665,11 @@ private void UserManualToolStripMenuItemClick(object sender, EventArgs e)
         {
             try
             {
-                Process.Start("http://git-extensions-documentation.readthedocs.org/en/master/");
+#if DEBUG
+                Process.Start("http://git-extensions-documentation.readthedocs.org/en/master");
+#else
+                Process.Start("http://git-extensions-documentation.readthedocs.org/");
+#endif
```

Comment:
Why not `https://git-extensions-documentation.readthedocs.io/en/release-3.4/`? We're on v3.4 branch here.
Later we can script it

Reply:
release/3.4 is better after we have released a version with many changes
However, we then get the merge problem, as we had this time. (This was one of the changes that was updated previously, I believe.)

Reply:
> I'd prefer the fidelity of the release branch, and deal with the merge when
it comes.

OK, I will change to only use release/3.4, no DEBUG but with a comment

Reply:
Merge at will

Reply:
I changed to just a comment and always getting the default
The reason is that the link will work also if we change the documentation structure, like remove the doc build some time in the future. Now you will just be redirected to the latest doc version instead of a 404. I tried to add redirect rules to overcome that, but the rules does not seem to allow that.
It is no major drawback to always show the latest version if the previous is available in the sidebar. (I set 3.1 and 3.2 as hidden though, the links work but are not displayed - no major changes there though.)